src/cdk/v2/destinations/tiktok_audience/rtWorkflow.yaml

```diff
@@ -14,14 +14,16 @@ steps:
 
   - name: successfulEvents
     template: |
-      $.outputs.transform#idx.output.({
-        "batchedRequest": .,
-        "batched": true,
-        "destination": ^[idx].destination,
-        "metadata": ^[idx].metadata,
-        "statusCode": 200
-      })[]
-
+      $.outputs.transform.map(function(transformedData,index){
```

Comment:
any reason you are using map function, you can do this without it, using native json template engine features